tests/UI/campaigns/functional/BO/09_shipping/02_preferences/02_carrierOptions/02_sortBy.js

```diff
@@ -0,0 +1,256 @@
+require('module-alias/register');
+
+// Import utils
+const helper = require('@utils/helpers');
+
+// Common tests login BO
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const preferencesPage = require('@pages/BO/shipping/preferences');
+const carriersPage = require('@pages/BO/shipping/carriers');
+const foHomePage = require('@pages/FO/home');
+const foProductPage = require('@pages/FO/product');
+const foCartPage = require('@pages/FO/cart');
+const foCheckoutPage = require('@pages/FO/checkout');
+
+// Import data
+const {Carriers} = require('@data/demo/carriers');
+const {DefaultAccount} = require('@data/demo/customer');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_shipping_preferences_carriersOptions_sortBy';
+
+// Import expect from chai
+const {expect} = require('chai');
+
+// Browser and tab
+let browserContext;
+let page;
+
+let numberOfCarriers = 0;
+
+/*
+Go to shipping > Carriers page
+Activate the 2 carriers 'My cheap carrier' and 'My light carrier'
+Go to shipping > preferences page
+Choose sort by 'Price'
+Go to FO and check the carriers sort
+Go back to BO and choose sort by 'Position'
+Go to FO and check the carriers sort
+Go back to BO > shipping > Carriers
+Disable the 2 carriers 'My cheap carrier' and 'My light carrier'
+ */
+describe('Update \'sort carriers by\' and check it in FO', async () => {
+  // before and after functions
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  it('should login in BO', async function () {
+    await loginCommon.loginBO(this, page);
+  });
+
+  it('should go to \'Shipping > Carriers\' page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToCarriersPageToEnable', baseContext);
+
+    await dashboardPage.goToSubMenu(
+      page,
+      dashboardPage.shippingLink,
+      dashboardPage.carriersLink,
+    );
+
+    const pageTitle = await carriersPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(carriersPage.pageTitle);
+  });
+
+  it('should reset all filters and get number of carriers in BO', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'resetFilterFirst', baseContext);
+
+    numberOfCarriers = await carriersPage.resetAndGetNumberOfLines(page);
+    await expect(numberOfCarriers).to.be.above(0);
+  });
+
+  const carriersNames = [
+    Carriers.cheapCarrier.name,
+    Carriers.lightCarrier.name,
+  ];
+  carriersNames.forEach((carrierName, index) => {
```

Comment:
```suggestion
  [
    Carriers.cheapCarrier.name,
    Carriers.lightCarrier.name,
  ].forEach((carrierName, index) => {
```

Reply:
I used the const `carrierNames` 2 times in the test( when I enable and disable carriers)
But I can delete `sortCarriersBy`-> done